### ISSUE TITLE
can run 1 file but not directory

### ISSUE THREAD

**Issue by reharik** [zinserjan/mocha-webpack#18]
Hi, I have just setup mocha-webpack.
When I specify a single file in the mocha-webpack.opts and run npm test, it runs the tests in the file but when I specify the parent directory it says 0 tests.  Below are the relevant files

`const path = require('path');
const webpack = require('webpack');
const HtmlWebpackPlugin = require('html-webpack-plugin');
var nodeExternals = require('webpack-node-externals');

const config = {
    // Gives you sourcemaps without slowing down rebundling
    output: {
        // sourcemap support for IntelliJ/Webstorm
        devtoolModuleFilenameTemplate: '[absolute-resource-path]',
        devtoolFallbackModuleFilenameTemplate: '[absolute-resource-path]?[hash]'
    },
    target: 'node', // in order to ignore built-in modules like path, fs, etc.
    externals: [nodeExternals()], // in order to ignore all modules in node_modules folder
    devtool: "cheap-module-source-map", // faster than 'source-map',
    module: {
        noParse: [],
        loaders: [
            { test: /\.jsx?$/, exclude: /(node_modules|bower_components)/, loader: 'babel-loader' },
            { test: /\.css$/, loader: 'style-loader!css-loader' },
            { test: /\.eot(\?v=\d+\.\d+\.\d+)?$/, loader: 'file' },
            // { test: /\.ttf(\?v=\d+\.\d+\.\d+)?$/, loader: "url?limit=10000&mimetype=application/octet-stream" },
            { test: /\.svg(\?v=\d+\.\d+\.\d+)?$/, loader: 'url?limit=10000&mimetype=image/svg+xml' },
            { test: /\.png$/, loader: 'url-loader', query: { mimetype: 'image/png' } },
            { test: /\.jpg$/, loader: 'url-loader', query: { mimetype: 'image/jpg' } },
            { test: /\.gif$/, loader: 'url-loader', query: { mimetype: 'image/gif' } },
            { test: /\.scss$/, loaders: ['style', 'css', 'sass'] }
        ]
    },
    plugins: [
        new webpack.optimize.OccurenceOrderPlugin(),
        new webpack.NoErrorsPlugin()
    ],
    sassLoader: {
        includePaths: [path.resolve(__dirname, './sass')]
    }
};

module.exports = config;`

mocha-webpack.opts

--colors
--webpack-config app/webpack.config-testing.js
--recursive
// -- no tests run
./app/tests/components
// -- 4 tests run 
./app/tests/components/ChapterTester.js

I'm just running 
    "test": "mocha-webpack ",

I feel like I'm really close, please let me know what you think
Thanks,
Raif

sorry formatting doesn't seem to be working :(

**Comment by reauv** [comment]
You should use three back-ticks for multi line code blocks, [as described here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting).

I am not sure why the recursive flag is not working, I always use a glob pattern like this:
`./app/tests/components/**/*.js`
Can you try if that works for you?

**Comment by reharik** [comment]
No, that doesn't seem to work either.  I'll dig in see what I can find.
R

**Comment by reharik** [comment]
ok, after some considerable pain ( for starters I started reading the compiled code, which was a bit strange, then I had to use the compiled code to get my console logs to show up ) I found the problem.
my file path 
./app/tests
it was trying to match 
app/tests
so everything was failing the regex because of the first two characters.
the single file of course didn't use the regex so no problem.

I guess I would suggest either normalizing the user input for them or making sure the docs call this out nice and clearly. 

Anyway, it's a great plug in.
Thanks,
R
